lib/kino_bumblebee/task_cell.ex

```diff
@@ -1161,10 +1174,8 @@ defmodule KinoBumblebee.TaskCell do
 
         frame = Kino.Frame.new()
 
-        form
-        |> Kino.Control.stream()
-        |> Kino.listen(fn %{data: %{audio: audio}} ->
-          Kino.Frame.render(frame, Kino.Markdown.new("Running..."))
+        Kino.listen(form, fn %{data: %{audio: audio}} ->
+          Kino.Frame.render(frame, Kino.Text.new("Running..."))
```

Comment:
Do we also want to wrap it in a conditional like `image`?

Reply:
Yess!